modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/ReplicatorUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.replicator;
+
+import static org.apache.ignite.internal.catalog.descriptors.CatalogIndexStatus.BUILDING;
+
+import org.apache.ignite.internal.catalog.CatalogService;
+import org.apache.ignite.internal.catalog.descriptors.CatalogIndexDescriptor;
+import org.apache.ignite.internal.catalog.descriptors.CatalogIndexStatus;
+import org.apache.ignite.internal.hlc.HybridTimestamp;
+import org.apache.ignite.internal.table.distributed.replication.request.ReadWriteReplicaRequest;
+import org.apache.ignite.internal.tx.TransactionIds;
+import org.jetbrains.annotations.Nullable;
+
+/** Auxiliary class. */
+class ReplicatorUtils {
+    /**
+     * Looks for the latest index with {@link CatalogIndexStatus#BUILDING} for the table, {@code null} if missing.
+     *
+     * <p>NOTE: It is expected that the method will be executed in the metastore thread so that the catalog does not change.</p>
```

Comment:
```suggestion
     * <p>NOTE: It is expected that the method will be executed in the metastore thread so that the catalog does not change concurrently.</p>
```

Reply:
fix it